src/api/Api.js

```diff
@@ -0,0 +1,31 @@
+import superagent from 'superagent';
+
+class Api {
```

Comment:
Since the functions are currently not sharing any data I think it's unusual.

If you want for some reason keep the class, it would be beneficial to

* not make the functions static and
* add the API base url as a parameter to the constructor.

Then you can export an instantiated API object.

Ex:
~~~ javascript
class Api {
  constructor(baseUrl) {
    this.baseUrl = baseUrl;
  }

  fetchX() {
    return superagent.get(`${this.baseUrl}/foo`);
  }
}

export default new Api(process.env.BASE_URL);
~~~

Or you drop the class altogether and take the API base url directly from the environment, which is what I would prefer.

Ex:
~~~ javascript
export function fetchX() {
  return superagent.get(`${process.env.BASE_URL}/foo`);
}
~~~